gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/proposeOffchainConfig.ts

```diff
@@ -231,6 +231,15 @@ export default class ProposeOffchainConfig extends SolanaCommand {
     const rawTx = await this.makeRawTransaction(this.wallet.publicKey)
     const startingPoint = new BN(this.flags.instruction || 0).toNumber()
 
+    const input = this.makeInput(this.flags.input)
```

Comment:
If there is a better way to get `randomSecret` like from `responses` I can refactor

Reply:
This won't work unfortunately, let me think on a better way